### DISCUSSION TITLE
Web interface for rclip?

### DISCUSSION THREAD

**Discussion by TheUnstoppablesAI** [yurijmikhalevich/rclip#119] [Ideas]
Is there a plan to implement or have an option for rclip? I've looked for other projects that use rclip to see if anyone has created a web interface and could not find one. Rclip is great but for some users who may not want to use terminal it's difficult. If there was a web interface then multiple PC's could access the rclip "server".

Great project by the way! I just found this yesterday and it works great on my Macbook Air M2! Thank you!

**Comment by yurijmikhalevich** [comment]
Hi @TheUnstoppablesAI! 👋 

Thank you for your kind words! I am glad you like rclip!

I am considering building a separate project providing a web UI powered by rclip. No timelines or promises, though, at this moment, since I am personally happy to use the terminal version.

I am trying to understand the direction the rclip project should take and your input will be helpful. If you have 15 minutes for a call to discuss rclip, send me an email at yurij@mikhalevi.ch, please :)
